bikeshed/caniuse/caniuse.py

```diff
@@ -13,6 +13,9 @@ def addCanIUsePanels(doc: t.SpecT) -> None:
     # for a term's feature.
     if not doc.md.includeCanIUsePanels:
         return
+    main = h.findAll("main", doc.body)[0]
```

Comment:
This'll throw if there's no main element.

```suggestion
    main = h.find("main", doc.body)
```